UnitySDK/Assets/ML-Agents/Examples/SharedAssets/Materials/ShaderOutline.shader

```diff
@@ -0,0 +1,88 @@
+﻿// Upgrade NOTE: replaced 'mul(UNITY_MATRIX_MVP,*)' with 'UnityObjectToClipPos(*)'
```

Comment:
We will need versions once we get into SRP-capable Unity versions (2018.1 and above) if we want users to be able to see this in LWRP, HDRP, or URP. This will be a mess until we standardize on an Editor version using SRP.